Changed/added some code

Dimitrios1117 opened the pull request:
Added some UI elements such as entering username between 1-10 characters
inclusive. Also, you can hit escape to pause the game and hit it again
to un pause it.

Dimitrios1117 committed:
Changed/added some code

Added some UI elements such as entering username between 1-10 characters
inclusive. Also, you can hit escape to pause the game and hit it again
to un pause it.

BCBandy commented:
Ok, BTW I was thinking the intake pause could just open the title screen
since they will do the same thing
On Nov 6, 2015 3:26 PM, "Dimitrios1117" notifications@github.com wrote:

> Added some UI elements such as entering username between 1-10 characters
> inclusive. Also, you can hit escape to pause the game and hit it again
> 
> ## to un pause it.
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/BCBandy/FamilyJewels/pull/1
> Commit Summary
> - Changed/added some code
> 
> File Changes
> - _M_ Family Jewels/Family Jewels/Family_Jewels.py
>   https://github.com/BCBandy/FamilyJewels/pull/1/files#diff-0 (137)
> - _M_ Family Jewels/Family Jewels/player.py
>   https://github.com/BCBandy/FamilyJewels/pull/1/files#diff-1 (6)
> 
> Patch Links:
> - https://github.com/BCBandy/FamilyJewels/pull/1.patch
> - https://github.com/BCBandy/FamilyJewels/pull/1.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/BCBandy/FamilyJewels/pull/1.

Dimitrios1117 committed:
Changed a lot of stuff

added save on death and score button on main menu.

Dimitrios1117 committed:
added menu title and top 10 scores

asads

Dimitrios1117 committed:
updated sorting to work as intended

Updated sorting

Dimitrios1117 committed:
Database

Added parse database saving and loading. Hopefully it works! Will need
to get the parse module to get it to work though.
